src/pages/ConnectPayment/components/AdyenCheckout.tsx

```diff
@@ -0,0 +1,238 @@
+import { colorsV2 } from '@hedviginsurance/brand'
+import {
+  getLocaleIsoCode,
+  useCurrentLocale,
+} from 'components/utils/CurrentLocale'
+import {
+  Scalars,
+  SubmitAdditionalPaymentDetialsMutationFn,
+  TokenizationChannel,
+  TokenizePaymentDetailsMutationFn,
+  useAvailablePaymentMethodsQuery,
+  useSubmitAdditionalPaymentDetialsMutation,
+  useTokenizePaymentDetailsMutation,
+} from 'data/graphql'
+import { match } from 'matchly'
+import { Spinner } from 'new-components/utils'
+import * as React from 'react'
+import { useHistory } from 'react-router'
+import { SpinnerWrapper } from './Spinner'
+
+const Wrapper = styled('div')`
+  position: relative;
+
+  @media (min-width: 801px) {
+    padding-right: 5rem;
+  }
+
+  .adyen-checkout__payment-method {
+    &,
+    &:first-child,
+    &:last-child {
+      border-radius: 3px;
+      border-top-left-radius: 3px;
+      border-top-right-radius: 3px;
+    }
+  }
+  .adyen-checkout__payment-method--selected {
+    background: ${colorsV2.offwhite};
+  }
+`
+
+export const AdyenCheckout = () => {
+  const availablePaymentMethods = useAvailablePaymentMethodsQuery()
+  const adyenCheckoutRef = React.useRef<HTMLDivElement>()
+  const [adyenLoaded, setAdyenLoaded] = React.useState(false)
+  const [tokenizePaymentMutation] = useTokenizePaymentDetailsMutation()
+  const [
+    submitAdditionalPaymentDetails,
+  ] = useSubmitAdditionalPaymentDetialsMutation()
+  const history = useHistory()
+  const currentLocale = useCurrentLocale()
+
+  const paymentMethodsResponse =
+    availablePaymentMethods.data?.availablePaymentMethods.paymentMethodsResponse
+
+  React.useEffect(() => {
+    if (!paymentMethodsResponse || !adyenLoaded) {
+      return
+    }
+
+    createAdyenCheckout({
+      currentLocale,
+      paymentMethodsResponse,
+      tokenizePaymentMutation,
+      submitAdditionalPaymentDetails,
+      history,
+    }).mount(adyenCheckoutRef.current)
+  }, [paymentMethodsResponse, adyenLoaded])
+
+  React.useEffect(mountAdyenJs(setAdyenLoaded), [])
+
+  React.useEffect(mountAdyenCss, [])
+
+  return (
+    <Wrapper>
+      {!adyenLoaded && (
+        <SpinnerWrapper>
+          <Spinner />
+        </SpinnerWrapper>
+      )}
+      <div
+        ref={adyenCheckoutRef as React.MutableRefObject<HTMLDivElement>}
+      ></div>
+    </Wrapper>
+  )
+}
+
+interface AdyenCheckoutProps {
+  currentLocale: string
+  paymentMethodsResponse: Scalars['PaymentMethodsResponse']
+  tokenizePaymentMutation: TokenizePaymentDetailsMutationFn
+  submitAdditionalPaymentDetails: SubmitAdditionalPaymentDetialsMutationFn
+  history: ReturnType<typeof useHistory>
+}
+
+const createAdyenCheckout = ({
+  currentLocale,
+  paymentMethodsResponse,
+  tokenizePaymentMutation,
+  submitAdditionalPaymentDetails,
+  history,
+}: AdyenCheckoutProps) => {
+  const locale = match([
+    ['sv_SE', 'sv-SE'],
+    ['en_SE', 'en-US'],
+    ['nb_NO', 'no-NO'],
+    ['en_NO', 'en-US'],
+  ])(getLocaleIsoCode(currentLocale))
```

Comment:
There is a hook for this somewhere that Gustav created `useLocalIsoCode` or similar

Reply:
Maybe its gone lost in a merge somewhere? Cant find anything for `use.*iso` :/